docs/self-managed/platform-architecture/overview.md

```diff
@@ -19,7 +19,11 @@ In this architecture, Camunda Platform 8 Self-Managed can be accessed as follows
 - Keycloak authentication: `https://camunda.example.com/auth`
 - Zeebe gateway: `grpc://zeebe.camunda.example.com`
 
-It's also possible to set up an Ingress for each component. For more details, visit our [combined and separated Ingress set up guide](../platform-deployment/helm-kubernetes/guides/ingress-setup.md).
+It's also possible to set up an Ingress for each component. For more details, visit our [combined and separated Ingress setup guide](../platform-deployment/helm-kubernetes/guides/ingress-setup.md).
```

Comment:
I'm not able to comment on the image [camunda-platform-8-self-managed-architecture-diagram-separated-ingress.png](https://github.com/camunda/camunda-platform-docs/pull/1712/files#diff-47b7fa0d350e0bddb595c248df6e4358555ac22f6860069fc47ef973d96631a9) itself, but just an aesthetic comment, following the same pattern we used, I'd suggest group the WM components the same way we have for Identity and Zeebe.

Reply:
@aabouzaid: So, I guess you mean to put Web Modeler and its Postgres database into one group?

Reply:
Or do you suggest to include Web Modeler's different sub-components (restapi, webapp, websockets)? I wanted to do that first, but it felt too cluttered.

Reply:
the second one.
since the sub-components (restapi, webapp, websockets) have their own deployments and resources and they should be visible on the top-level diagram.

we can make it like the identity section.